src/browser-locale/private/getBrowserLocale.js

```diff
@@ -0,0 +1,64 @@
+/* global TERRA_AGGREGATED_LOCALES */
```

Comment:
Could we move this logic into application base?
```
./application-base
  /private
      getBrowserLocale
```
You said it shoudln't be consumed anywhere else right?

Reply:
That makes sense to me: https://github.com/cerner/terra-application/pull/13/commits/a9c1c89a5086c5d798f6377f507de1f3b8c9e2f0